common-features/input-validation.md

```diff
@@ -735,6 +741,12 @@ The sliders are, effectively, numeric inputs in terms of behavior and what data
     <TelerikButton ButtonType="@ButtonType.Submit">Submit</TelerikButton>
 </EditForm>
 
+<style>
+    .k-slider{
+        height: 60px
```

Comment:
If the purpose of this CSS rule is to increase the space between the Sliders and the labels, there are better ways to do that, which do not include our internal CSS classes - for example, add margins to the custom labels.